### PR TITLE
Make output_root a non parameter.

### PR DESCRIPTION
**Pull request by HassanJaveed84**
@brianhw @mulby this task does not use output_root, so made this a non-param on Brian's suggestion.

### CODE REVIEW

**Review by mulby — APPROVED** [review]
Ah, right, because we just use the warehouse path to generate the output path.

:+1:

File: edx/analytics/tasks/enrollments.py [hunk 1]
```diff
@@ -40,6 +40,8 @@ class CourseEnrollmentEventsTask(
     # FILEPATH_PATTERN should match the output files defined by output_path_for_key().
     FILEPATH_PATTERN = '.*?course_enrollment_events_(?P<date>\\d{4}-\\d{2}-\\d{2})'
 
+    output_root = ''
```
Comment (brianhw) [inline edx/analytics/tasks/enrollments.py:44]: It would be good to have a comment on this, noting why it's here.

**Comment by brianhw** [comment]
Add a comment, and this should be good to go. 👍

**Comment by brianhw** [comment]
LGTM 👍

**Review by brianhw — APPROVED** [review]
LGTM. 👍   (Trying again to approve.)

**Comment by HassanJaveed84** [comment]
Forgot to squash before merging 😞

**Comment by brianhw** [comment]
@HassanJaveed84 This seems to have broken master's unit tests.  Can you take a look?

**Pull request merged**